packages/fondue/src/components/Tooltip/Tooltip.tsx

```diff
@@ -37,21 +37,22 @@ const formatTooltipText = (text: string) => {
 };
 
 type TimeoutRef = {
-    current: NodeJS.Timeout | null;
+    current?: number;
```

Comment:
my bad! I have always typed it to`ReturnType<typeof setTimeout>` but since this only executes with the browser timer, and not Node, and using the`window` namespace, the `number` is fine :)